opm/parser/eclipse/EclipseState/Schedule/Well.hpp

```diff
@@ -112,6 +117,7 @@ namespace Opm {
         std::shared_ptr<DynamicState<WellProductionProperties> > m_productionProperties;
         std::shared_ptr<DynamicState<WellInjectionProperties> > m_injectionProperties;
         std::shared_ptr<DynamicState<WellPolymerProperties> > m_polymerProperties;
+        std::shared_ptr<DynamicState<WellSolventProperties> > m_solventProperties;
```

Comment:
As long as the solvent is only a single number I would completely remove the `WellSolventProperties` structure and just use:
```C++
std::shared_ptr<DynamicState<double> > m_solventProperties;
```